### PR TITLE
Update images that use ubuntu-slim base image to :0.6

### PR DESCRIPTION
**Pull request by ixdy**
**What this PR does / why we need it**: `ubuntu-slim:0.4` is somewhat old, being based on Ubuntu 16.04, whereas `ubuntu-slim:0.6` is based on Ubuntu 16.04.1.

**Special notes for your reviewer**: I haven't pushed any of these images yet, so I expect all of the e2e builds to fail. If we're happy with the changes, I can push the images and then re-trigger tests.

**Release note**:

```release-note
NONE
```

cc @aledbf as FYI

### CODE REVIEW

File: cluster/addons/fluentd-elasticsearch/kibana-image/Makefile [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 .PHONY:	build push
 
-TAG = v4.6.1
+TAG = v4.6.1-1
```
Comment (timstclair) [inline cluster/addons/fluentd-elasticsearch/kibana-image/Makefile]: Why not update the patch version? Does 4.6.1 correspond to something?

Comment (ixdy) [inline cluster/addons/fluentd-elasticsearch/kibana-image/Makefile]: yeah, it appears to correlate with the Kibana version - https://github.com/ixdy/kubernetes/blob/1c2ea2808012dcea8e785f14c0e155ed66c2808a/cluster/addons/fluentd-elasticsearch/kibana-image/Dockerfile#L22

Alternately, there is a Kibana 4.6.3 we could upgrade to, along with updating the base image. (It's actually up to 5.1.1, but 5.0 had some breaking changes which may or may not affect us.)

**Comment by Crassirostris** [comment]
@ixdy I'm happy with fluentd changes, thanks!

File: cluster/addons/fluentd-elasticsearch/kibana-image/Makefile (hunk 1 above)
Comment (Crassirostris) [inline cluster/addons/fluentd-elasticsearch/kibana-image/Makefile]: Exactly. Moreover, kibana 5.X is intended to work with ES 5.X, which contains a lot of compatible changes.

I will update it one day, but it requires some testing and maybe e2e tests changes.

**Comment by ixdy** [comment]
Pushed images.

@k8s-bot test this

**Pull request merged**